profiles/urls.py

```diff
@@ -8,4 +8,5 @@
     path('update-summary/', views.UpdateSummary.as_view(), name='update-summary'),
     path('add-skill/<slug:skill>', views.AddSkill.as_view(), name='add_skill'),
     path('remove-skill/<slug:skill>', views.RemoveSkill.as_view(), name="remove_skill"),
+    path('add-responsibility', views.AddResponsibility.as_view(), name="add_responsibility")
```

Comment:
Can we add trailing / to this and the above 2 URL's, just as best practice and to be consistent

Reply:
Done

Reply:
9 & 10 too?? Sorry nit picking lol

Reply:
Ok that's done now